spec/rocket_chat/messages/group_spec.rb

```diff
@@ -104,14 +104,14 @@
   end
 
   describe '#list_all' do
-    let(:room1) do
+    let(:room_one) do
```

Comment:
although I get why the lint exists, in the this case it's not really an issue. I've already "fixed" this by just allowing up to 2 indexed let variables. But this is fine too